content/en/observability_pipelines/configurations.md

```diff
@@ -1,32 +1,27 @@
 ---
-title: Vector Configurations
+title: Configurations
 kind: Documentation
 aliases:
   - /integrations/observability_pipelines/vector_configurations/
+  - /observability_pipelines/vector_configurations/
```

Comment:
can we replace this with just /configurations/?

Reply:
This is an alias, meaning that it's just redirecting the page that used to live at `/observability_pipelines/vector_configurations/` to /observability_pipelines/configurations/